corehq/apps/app_manager/views/download.py

```diff
@@ -249,6 +249,10 @@ def resolve_path(path):
         if request.app.copy_of:
             if request.META.get('HTTP_USER_AGENT') == 'bitlybot':
                 raise Http404()
+            elif path in ('CommCare.jad', 'CommCare.jar'):
+                request.app.create_jadjar(save=True)
+                request.app.save()
```

Comment:
Also I think you need `request.app.save(increment_version=False)` or else the version number of the build will change